frontend/packages/console-shared/src/components/pod/PodRing.tsx

```diff
@@ -69,20 +69,30 @@ const PodRing: React.FC<PodRingProps> = ({
   return (
     <Split>
       <SplitItem>
-        <div className="odc-pod-ring">
+        <div
+          className={`odc-pod-ring${
+            resourceObj.status.availableReplicas ? '' : ' odc-pod-ring--scaled'
+          }`}
+        >
           <PodStatus
             standalone
             data={pods}
-            subTitle={
-              resourceObj.spec.replicas !== resourceObj.status.availableReplicas
-                ? !resourceObj.spec.replicas
-                  ? `pods`
-                  : `scaling to ${resourceObj.spec.replicas}`
-                : resourceObj.spec.replicas > 1 || resourceObj.spec.replicas === 0
-                ? 'pods'
-                : 'pod'
+            {...(resourceObj.status.availableReplicas
+              ? {
+                  subTitle:
+                    resourceObj.spec.replicas !== resourceObj.status.availableReplicas
+                      ? !resourceObj.spec.replicas
+                        ? `pods`
+                        : `scaling to ${resourceObj.spec.replicas}`
+                      : resourceObj.spec.replicas > 1 || resourceObj.spec.replicas === 0
+                      ? 'pods'
+                      : 'pod',
+                }
+              : {})}
```

Comment:
@vikram-raj Please make this logic more readable. You can create a method and return the string that's needed for subtitle and title or empty string/undefined if no subtitle/title is needed.